src/components/editor-toolbar/index.jsx

```diff
@@ -49,6 +50,9 @@ const EditorToolbar = ( { className } ) => {
 	}, [] );
 	const { setIsInserterOpened } = useDispatch( editorStore );
 
+	useModalize( isInserterOpened );
+	useModalize( isBlockInspectorShown );
```

Comment:
Mark content outside of the block inserter and block inspector as inert, so that screen readers cannot read the content aloud when visually obscured by a modal.